spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/SpatialBatchNormalization.scala

```diff
@@ -62,12 +64,10 @@ class SpatialBatchNormalization(
   val weightAndBias = new TensorMMap(Array(nOutput * 2))
   val gradWeightAndBias = new TensorMMap(Array(nOutput * 2))
 
-  var scaleFactor: Float = 1.0f
-  var biasFactor: Float = 1.0f
-
   private val runningMeanScaled = Tensor[Float].resizeAs(runningMean.dense)
   private val runningVarianceScaled = Tensor[Float].resizeAs(runningVariance.dense)
 
+  // the blank shoud be here, otherwise the runningVarianceScaled will be a method
```

Comment:
what does this mean

Reply:
like the comment. otherwise scala compiler will think this is a method.